finance-admin-api/api/request_report.go

```diff
@@ -61,9 +61,15 @@ func (s *Server) generateAndUploadReport(ctx context.Context, reportRequest shar
 				FromDate: reportRequest.FromDateField,
 				ToDate:   reportRequest.ToDateField,
 			}
+		case shared.ReportAccountTypeAgedDebtByCustomer:
+			query = &db.AgedDebtByCustomer{}
 		}
 	}
 
+	if query == nil {
```

Comment:
Instead of putting this here, would it be cleaner to include it as a default to the switch statement(s)?